harness/determined/common/experimental/determined.py

```diff
@@ -26,20 +62,62 @@ def __init__(
         user: Optional[str] = None,
     ):
         self._session = Session(master, user)
+        self._auth = auth.Authentication.instance()
+        self._configuration = Configuration()
+
+        # Remove trailing '/' character for Swagger
+        if self._session._master[-1] == "/":
+            self._configuration.host = self._session._master[:-1]
+        else:
+            self._configuration.host = self._session._master
+        self._configuration.username = self._auth.token_store.get_active_user()
```

Comment:
I don't think configuration needs to be saved to `self`

Reply:
removed